wp-content/plugins/core/src/Blocks/Types/Stats/Stats_Model.php

```diff
@@ -0,0 +1,21 @@
+<?php
+declare( strict_types=1 );
+
+namespace Tribe\Project\Blocks\Types\Stats;
+
+use Tribe\Project\Blocks\Types\Base_Model;
+use Tribe\Project\Templates\Components\blocks\stats\Stats_Block_Controller;
+
+class Stats_Model extends Base_Model {
+
+	public function get_data(): array {
+		return [
+			Stats_Block_Controller::LAYOUT           => $this->get( Stats::LAYOUT, Stats::LAYOUT_STACKED ),
+			Stats_Block_Controller::CONTENT_ALIGN    => $this->get( Stats::CONTENT_ALIGN, Stats::CONTENT_ALIGN_LEFT ),
+			Stats_Block_Controller::DISPLAY_DIVIDERS => $this->get( Stats::DISPLAY_DIVIDERS, true ),
+			Stats_Block_Controller::TITLE            => $this->get( Stats::TITLE, '' ),
+			Stats_Block_Controller::DESCRIPTION      => $this->get( Stats::DESCRIPTION, '' ),
+			Stats_Block_Controller::STATS            => $this->get( Stats::STATS, [] ),
```

Comment:
I ran into an issue getting repeater values this way in the Tabs Block. It returned the repeater count, rather than the array of values.

Regardless, it might be nice to setup a "Statistic Template Model" here so that the data provided to the Block and respective Components has the correct format/values. Here's a set of examples from the Tabs Block:
* You can see how I set that up in the Tabs component [here](https://github.com/moderntribe/square-one/pull/557/files#diff-438d1ba3de385ebf852fd24c46543ea3R53-R65).
* The Tab Template Model is [here](https://github.com/moderntribe/square-one/blob/epic/fidgety-feet/wp-content/plugins/core/src/Templates/Models/Tab.php).
* Then, you can see I'm denoting that the incoming argument type should be an array of those Tab Models [here](https://github.com/moderntribe/square-one/pull/557/files#diff-421120537c1c477ee0b791f1dc53d242R25-R28).
* Finally, you can see how I reference each Model Object's properties (rather than needing to reference array keys) [here](https://github.com/moderntribe/square-one/pull/557/files#diff-421120537c1c477ee0b791f1dc53d242R139) (and elsewhere in this file).

Reply:
@dpellenwood this should be corrected with a recent update to the Base_Model on how we are getting fields. It shouldn't return the number of repeaters any longer.